Second_semestr/Homework_7/List(T)/List(T)/List.cs

```diff
@@ -0,0 +1,247 @@
+﻿using System;
+using System.Collections;
+
+namespace ListNamespace
+{
+    /// <summary>
+    /// singly linked list
+    /// </summary>
+    public class List<T> : IList<T>, IEnumerable
```

Comment:
Извините:) я чуток не понял, вместо IEnumerable что то другое должно было
быть?


2016-05-16 20:12 GMT+03:00 Yurii Litvinov <notifications@github.com>:

> In Second_semestr/Homework_7/List(T)/List(T)/List.cs
> <https://github.com/AlbertMukhammadiev/University/pull/34#discussion_r63388284>
> :
>
> > @@ -0,0 +1,247 @@
> > +﻿using System;
> > +using System.Collections;
> > +
> > +namespace ListNamespace
> > +{
> > +    /// <summary>
> > +    /// singly linked list
> > +    /// </summary>
> > +    public class List<T> : IList<T>, IEnumerable
>
> надо IEnumerable (кстати, узнайте, чем он отличается от IEnumerable), и
> надо, чтобы даже IList реализовывал IEnumerable
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AlbertMukhammadiev/University/pull/34/files/c71f04df3c8211e458cf57ccba6838bb13192725#r63388284>
>